carma-messenger-config/chevrolet_tahoe_2018/docker-compose.yml

```diff
@@ -35,8 +33,6 @@ services:
     network_mode: host
     volumes_from:
       - container:carma-config:ro
-    environment:
-      - CYCLONEDDS_URI=/opt/carma/vehicle/calibration/telematics/cyclone_config.xml
```

Comment:
We still set the `CYCLONEDDS_URI` in our [lexus_rx_450h_2019](https://github.com/usdot-fhwa-stol/carma-config/blob/develop/lexus_rx_450h_2019/docker-compose.yml) and [chrysler_pacifica_ehybrid_s_2019](https://github.com/usdot-fhwa-stol/carma-config/blob/develop/chrysler_pacifica_ehybrid_s_2019/docker-compose.yml) docker-compose files in carma-config. Should we remove these as well?

Reply:
Yes, good point - those would be good to remove for the time being as well.

Reply:
Put up a PR to remove this from the fusion and lexus as well. https://github.com/usdot-fhwa-stol/carma-config/pull/445/files
Pacifica doesn't set this.